phpBB/phpbb/controller/helper.php

```diff
@@ -184,6 +184,8 @@ public function route($route, array $params = array(), $is_amp = true, $session_
 	* @param string $message The error message
 	* @param int $code The error code (e.g. 404, 500, 503, etc.)
 	* @return Response A Response instance
+	*
+	* @deprecated 3.1.3 (To be removed: 3.3.0) Use exceptions instead.
```

Comment:
I meant in the comment, like we do in this cases: https://github.com/phpbb/phpbb/blob/develop/phpBB/includes/functions.php#L3212
And also maybe explain which ones to use...http_exception, runtime_exception, phbb_exception, etc. in which cases, like which one for AJAX or non-AJAX ?